server/rpc.go

```diff
@@ -135,7 +136,7 @@ func verifyOrchestratorReq(orch Orchestrator, addr ethcommon.Address, sig []byte
 		glog.Error("orchestrator req sig check failed")
 		return fmt.Errorf("orchestrator req sig check failed")
 	}
-	return nil
+	return orch.CheckCapacity("")
```

Comment:
> this capacity check would never return an error as long as the transcode loop for the empty manifestID is active right

Correct, just as it would not return an error if the transcode loop was active for any non-empty ManifestID. So it's not bypassing any checks beyond the possibility of hijacking the loop for a somehow ill-formed ManifestID.

Although this is mostly harmless, it's still another example of why I'd prefer not to use the Manifest ID as an index to the transcoding loop as mentioned in https://github.com/livepeer/go-livepeer/issues/695 .